requirements/constraints.txt

```diff
@@ -68,3 +68,13 @@ pennsieve>=999999999
 
 # User alternative e.g. parametrized fixture or request.getfixturevalue(.)
 pytest-lazy-fixture>=999999999
+
+# avoid downgrades of openapi-spec-validator related libraries
+referencing<=0.35.1
+
+# moto==5.0.21 broke `get_ec2_instance_capabilities`
+moto<5.0.21
+
+# Pin boto3<1.36.0 till the following is addressed https://github.com/boto/botocore/issues/2308
+# when removing this pin, also update the aws cli inside ci/github/helpers/install_aws_cli_v2.bash
+boto3<1.36.0
```

Comment:
did you create a maintenance issue for this?

Reply:
are we having a problem with this? this is marked as a feature request and is only happening in some cases.

Reply:
Please red though the linked issue. They can label it however they please, but my understanding is that they are missing a header in the requests. For me this is a bug, since AWS now requires it.